documentation/modules/ROOT/pages/crafting-cluster-telco-related-infra-operators-configs.adoc

```diff
@@ -93,11 +92,91 @@ spec:
   remediationAction: inform
   sourceFiles:
     - fileName: DisableSnoNetworkDiag.yaml
-      policyName: group-policies
+      policyName: "group-policy"
```

Comment:
I can't seem to comment on the lines above - for the spec.bindingRules, shouldn't we also make sure to include the label(s) that we are using in the hub templating, just as it was updated in the PGT below.